packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx

```diff
@@ -0,0 +1,146 @@
+                    <td css={headerCellStyle}>{t('id')}</td>
+                    <td css={contentCellStyle}>
+                        <a
+                            href={`${PATH_PROCESSES}/${subscriptionDetailProcess.processId}`}
+                        >
+                            {subscriptionDetailProcess.processId}
+                        </a>
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+                <tr>
+                    <td css={emptyCellStyle}></td>
+                    <td css={headerCellStyle}>{t('status')}</td>
+                    <td css={contentCellStyle}>
+                        <WFOProcessStatusBadge
+                            processStatus={subscriptionDetailProcess.lastStatus}
+                        />
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+                <tr>
+                    <td css={emptyCellStyle}></td>
+                    <td css={headerCellStyle}>{t('startedAt')}</td>
+                    <td css={contentCellStyle}>
+                        {parseDateToLocaleDateTimeString(
+                            parseDate(subscriptionDetailProcess.startedAt),
+                        )}
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+                <tr>
+                    <td css={emptyCellStyle}></td>
+                    <td css={lastHeaderCellStyle}>{t('startedBy')}</td>
+                    <td css={lastContentCellStyle}>
+                        {subscriptionDetailProcess.createdBy}
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+            </table>
+        </div>
+    );
+};
+
+interface WfoRenderSubscriptionProcess {
+    subscriptionDetailProcess: SubscriptionDetailProcess;
+}
+const WfoRenderSubscriptionProcess = ({
+    subscriptionDetailProcess,
+}: WfoRenderSubscriptionProcess) => {
+    const { timeLineStyle, workflowTargetStyle } =
+        useWithOrchestratorTheme(getStyles);
+
+    return (
+        <EuiComment
+            username={subscriptionDetailProcess.workflowTarget ?? ''}
+            timelineAvatarAriaLabel={subscriptionDetailProcess.workflowName}
+            timelineAvatar={
+                <EuiAvatar name={subscriptionDetailProcess.workflowTarget} />
+            }
+        >
+            <div css={timeLineStyle}>
+                <EuiText css={workflowTargetStyle}>
+                    {upperCaseFirstChar(
+                        subscriptionDetailProcess.workflowTarget,
+                    )}
+                </EuiText>
+                <EuiText>{subscriptionDetailProcess.workflowName}</EuiText>
+            </div>
+
+            <WfoProcessCard
+                subscriptionDetailProcess={subscriptionDetailProcess}
+            />
+        </EuiComment>
+    );
+};
+
+interface WfoProcessesTimelineProps {
```

Comment:
Maybe move to its own file within this subscription folder

Reply:
Well.. the processTimeline is in the right file but the other components perhaps